Sources/Latte+UniversalLink.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import UIKit
+
+@available(iOS 13.0, *)
+public extension Latte {
+    static func handleUniversalLink(
+        context: UINavigationController,
+        authgear: Authgear,
+        customUIEndpoint: String,
+        userActivity: NSUserActivity
+    ) -> Bool {
+        guard userActivity.activityType == NSUserActivityTypeBrowsingWeb,
+              let incomingURL = userActivity.webpageURL,
+              let components = NSURLComponents(url: incomingURL, resolvingAgainstBaseURL: true) else {
+            return false
+        }
+        guard let path = components.path,
+              let params = components.queryItems else {
+            return false
+        }
+        switch path {
+        case _ where path.hasSuffix("/reset_link"):
+            Latte.handleResetLink(
+                context: context,
+                authgear: authgear,
+                customUIEndpoint: customUIEndpoint,
+                query: params
+            )
+            return true
+        default:
+            return false
+        }
+    }
+
+    private static func handleResetLink(
```

Comment:
This should be a public function on `Latte`, and return a `Latte.Handle<Void>` to completion handler.